.github/workflows/new-release-tag-docker.yml

```diff
@@ -47,6 +63,34 @@ jobs:
             Fill in with release drafter information manually for now, then publish.
           draft: true
           prerelease: false
+      - name: Upload Release Asset
+        id: upload-release-asset
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ./build/defectdojo-${{ steps.package-helm-chart.outputs.chart_version }}.tgz
+          asset_name: defectdojo-${{ steps.package-helm-chart.outputs.chart_version }}.tgz
+          asset_content_type: application/tar+gzip
+      - name: Update Helm repository index
+        id: update-helm-repository-index
+        run: |
+          git config --global user.name "${{ env.GIT_USERNAME }}"
+          git config --global user.email "${{ env.GIT_EMAIL }}"
+          git remote update
+          git fetch --all
+          git checkout gh-pages
+          git pull
+          if [ ! -f ./index.yaml ]; then
+            helm repo index ./build --url "https://github.com/DefectDojo/django-DefectDojo/releases/download/${{ github.event.inputs.release_number }}/"
```

Comment:
Do you think there is a eventually better way than hard-coding the URL string?

Reply:
I've just pushed a modification to try to use environment variables but I can't check if the resulting URL is correct (I don't see the corresponding workflow in the Github interface...).

Reply:
Seems to be good to me.